src/plugins/image_display/ImageDisplay.cc

```diff
@@ -230,174 +278,6 @@ void ImageDisplay::OnRefresh()
   this->TopicListChanged();
 }
 
-/////////////////////////////////////////////////
-void ImageDisplay::UpdateFromRgbInt8()
```

Comment:
Can the removed methods be removed from `ImageDisplay.hh` as well?

Reply:
good catch. Done. 25eaf0c1d6233024dee32883fdb0447c7119de68